packages/qwik-auth/src/index.ts

```diff
@@ -1,5 +1,6 @@
 import { Auth, skipCSRFCheck } from '@auth/core';
-import type { AuthAction, AuthConfig, Session } from '@auth/core/types';
+import type { AuthAction, Session } from '@auth/core/types';
+import type { AuthConfig } from '@auth/core/index';
```

Comment:
is this correct?

Reply:
There is nothing wrong with my local. However, I will check it again after I finish to fix another one